weaknesses/MASVS-RESILIENCE/MASWE-0098.md

```diff
@@ -2,17 +2,43 @@
 title: App Virtualization Environment Detection Not Implemented
 id: MASWE-0098
 alias: app-virtualization-detection
-platform: [android, ios]
+platform: [android]
 profiles: [R]
 mappings:
-  masvs-v2: [MASVS-RESILIENCE-1]
-
-draft:
-  description: runs as a so-called "clone app"
-  topics:
-  - detection in place
-  - Effectiveness Assessment (e.g. bypassing the detection)
+  masvs-v1: [MSTG-RESILIENCE-3, MSTG-RESILIENCE-4, MSTG-RESILIENCE-5, MSTG-RESILIENCE-6]
```

Comment:
```suggestion
  masvs-v1: [MSTG-RESILIENCE-4, MSTG-RESILIENCE-6]
```
I don't see what static-analysis has to do with this, but feel free to counter-argument :)

Emulator (5) also seems irrelevant. The goal of the test is to detect the encapsulation in an app virtualization app, which may or may not be running on an emulator.

Reply:
`I don't see what static-analysis has to do with this` None of these MSTG-IDs referring to static-analysis. @TheDauntless can you elaborate more on that?

Emulator detection is a stretch but I've encountered a few virtualizers like [TWOYI] (https://github.com/twoyi/twoyi). Yes, it's an emulator but it perfectly fits into the Android virtualization definition. Also, these MSTG-IDs are here only for legacy reasons.

Reply:
I think I confused v1 and v2. My bad.